extensions/amp-story/1.0/test/validator-amp-story-animations-error.out

```diff
@@ -75,4 +74,4 @@ amp-story/1.0/test/validator-amp-story-animations-error.html:54:12 The attribute
 |        </amp-story-page>
 |    </amp-story>
 |  </body>
-|  </html>
+|  </html>
```

Comment:
nit: newlines here and in other `.out` files. (--update-tests flag doesn't add it but you can add it manually)